### PR TITLE
fix #75474 hyundai.dacar-spb.ru

### LINKED ISSUE #75474: hyundai.dacar-spb.ru
### Issue URL (Annoyance)
 [https://hyundai.dacar-spb.ru/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fhyundai.dacar-spb.ru%2F)
### Comment
Username: `@Rakleed`
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://yandex.ru/maps/org/hyundai_diler_karsavto/139052700237/?display-text=%D0%90%D0%B2%D1%82%D0%BE%D1%81%D0%B0%D0%BB%D0%BE%D0%BD&ll=30.385404%2C59.888479&mode=search&sctx=ZAAAAAgBEAAaKAoSCZig6ZkcYj5AEV5bR26w8U1AEhIJNe%2FjQzvwdj8RWmDmINs7Wj8oCjgAQN%2BuB0gBVc3MzD5YAGoCcnVwAJ0BzcxMPaABAKgBAL0BIEyLrsIBPo%2Bx7dwOj%2Bjc2F7nteTZqQaYvcj4F82kv4GGBMv9k5%2FGBo%2F7mveCBNS72r30BZK00ajuA%2F2JrMV%2F15zgkLwD6gEA8gEA%2BAEAggIS0JDQstGC0L7RgdCw0LvQvtC9igIJMTg0MTA1MzIy&sll=30.385404%2C59.888479&sspn=0.008727%2C0.002495&text=%D0%90%D0%B2%D1%82%D0%BE%D1%81%D0%B0%D0%BB%D0%BE%D0%BD&z=17.82
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Hide your Referrer from third-parties (hyundai.dacar-spb.ru), <br>Hide your User-Agent ()
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by stanislav-atr**
#75474

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -124,6 +124,9 @@ giardiniblog.it#$##bio_ep { display: none !important; }
 giardiniblog.it#$##bio_ep_bg { display: none !important; }
 puntocellulare.it#$?##pushbox { remove: true; }
 puntocellulare.it#%#//scriptlet('set-cookie', 'ilike', '1')
+hyundai.dacar-spb.ru#$##action-modal { display: none !important; }
+hyundai.dacar-spb.ru#$#body { overflow: visible !important; }
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/popups.txt:128]: You forgot about the backdrop.

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -124,6 +124,9 @@ giardiniblog.it#$##bio_ep { display: none !important; }
...
+hyundai.dacar-spb.ru##body > div[data-target="#credit-modal"]
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/popups.txt:129]: why not `dacar-spb.ru##.callback-bt`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

**Comment by stanislav-atr** [comment]
@Sergey-Lyapin, @lolka4toli , i looked into it and i am pretty sure that's a scam website.
It's fishing for www.hyundai-dacar.ru/ users: almost identical domain name, poor layout and design, prices are 50% of market and it is not mentioned as a dealer at www.hyundai.ru/find-dealer. I think we should block it.

**Comment by Sergey-Lyapin** [comment]
> @Sergey-Lyapin, @lolka4toli , i looked into it and i am pretty sure that's a scam website.
> It's fishing for [www.hyundai-dacar.ru/](http://www.hyundai-dacar.ru/) users: almost identical domain name, poor layout and design, prices are 50% of market and it is not mentioned as a dealer at [www.hyundai.ru/find-dealer](http://www.hyundai.ru/find-dealer). I think we should block it.

@stanislav-atr
and It has 2640 visitors/month =)

**Comment by lolka4toli** [comment]
https://yandex.ru/maps/org/dakar_ofitsialny_diler_hyundai/169345254877/?ll=30.483644%2C59.929906&z=15

image

**Pull request closed without merging**